incubator/binding-openapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/internal/generator/http/proxy/OpenApiHttpProxyConfigGenerator.java

```diff
@@ -0,0 +1,656 @@
+/*
```

Comment:
Let's remove this one as we already have the gist in the composite binding adapter now.